bin/oneoff/backfill_data/backfill_address_line1_in_schools

```diff
@@ -33,14 +33,16 @@ failure_count = 0
 
 open(err_file, 'w') do |err|
   CSV.foreach(tmp_file.path, {col_sep: "\t", headers: true, quote_char: "\x00"}) do |row|
-    begin
-      school = School.find_by_id(row['NCESSCH'].to_i.to_s)
-      school[:address_line1] = row['LSTREE']
-      school.save!
-      success_count += 1
-    rescue
-      err << row
-      failure_count += 1
+    school = School.find_by_id(row['NCESSCH'].to_i.to_s)
+    unless school.nil?
```

Comment:
Note you can also do `next unless school` (a common pattern). Not a big deal either way.

Reply:
Updated.